ee/models/organization_resource_access.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+from posthog.models.organization import Organization
+
+
+class OrganizationResourceAccess(models.Model):
+    class AccessLevel(models.IntegerChoices):
+        """Level for which a role or user can edit or view resources"""
+
+        CAN_ONLY_VIEW = 21, "Can only view"
+        CAN_ALWAYS_EDIT = 37, "Can always edit"
+        DEFAULT_VIEW_ALLOW_EDIT_BASED_ON_ROLE = 25, "Default view unless role grants access"
```

Comment:
what does this mean?

Reply:
Members can be view only or edit only and then we won't need to check if they belong to any feature flag designated roles. Otherwise that level denotes "custom feature flag role" access which is for when you want the user to be view only for everything but have access to certain flags. It communicates more clearly to customers that they can set strictly "view only" or "edit always" or a third custom "per flag access" for their users and roles. 

Additionally we now only have to check if a user belongs to the "custom access" role before checking what feature flags x roles they're connected to instead of always having to do that with "view only" permissions, so it should save us an extra search

Reply:
removed